### ISSUE TITLE
Update status types

### ISSUE THREAD

**Issue by katz95** [spotify/backstage#724]
## 🗒 General
Hi!! Would love some help updating our status types. To make them more accessible and uniform, we've decided that statuses will be indicated by a colored dot *and* text. 

<img width="695" alt="Screenshot 2020-05-05 at 4 58 24 PM" src="https://user-images.githubusercontent.com/61153904/81080946-a681ac00-8ef1-11ea-90d5-1a2e10e35792.png">

## 💻 Usage
Statuses ought to be indicated by a dot and text. We advise against just displaying a dot (for accessibility and communication purposes).

## 📐 Specs
<img width="486" alt="Screenshot 2020-05-05 at 5 03 54 PM" src="https://user-images.githubusercontent.com/61153904/81081613-6a028000-8ef2-11ea-90cd-2494b58659ba.png">

**Comment by katz95** [comment]
Please update the storybook as well! 🙏 

[](http://storybook.backstage.io/?path=/story/status--default)

[](http://storybook.backstage.io/?path=/story/sortable-table--default)

**Comment by freben** [comment]
8px diameter right? And are these dots not to be specified in em terms (relative to the font size)?

**Comment by katz95** [comment]
@freben Ahh, that's a good point! Yes they should be relative to the font size

**Comment by adaszyn** [comment]
@katz95 Here's a PR with the changes :) https://github.com/spotify/backstage/pull/811

**Comment by stefanalund** [comment]
Closed in #811
